crates/graphql/src/schema/objects/nft.rs

```diff
@@ -275,3 +279,43 @@ If no value is provided, it will return XSmall")))]
             .map_err(Into::into)
     }
 }
+
+#[derive(Debug, Clone)]
+pub struct NftCount {
+    creators: Vec<PublicKey<NftCreator>>,
+}
+
+impl NftCount {
+    #[must_use]
+    pub fn new(creators: Vec<PublicKey<NftCreator>>) -> Self {
+        Self { creators }
+    }
+}
+
+#[graphql_object(Context = AppContext)]
+impl NftCount {
+    fn total(&self, context: &AppContext) -> FieldResult<i32> {
+        let conn = context.db_pool.get()?;
+        let creators: Vec<String> = self.creators.clone().into_iter().map(Into::into).collect();
+        let count = queries::nft_count::total(&conn, creators)?;
+        Ok(count.try_into()?)
+    }
+
+    #[graphql(arguments(auction_houses(description = "a list of auction house public keys")))]
+    fn listed(
+        &self,
+        context: &AppContext,
+        auction_houses: Vec<PublicKey<AuctionHouse>>,
```

Comment:
```suggestion
        auction_houses: Option<Vec<PublicKey<AuctionHouse>>>,
```

Optional here too. remember about boxing the query then adding more filters if is_some can see queries::metadata for example.